src/System.Private.ServiceModel/tools/scripts/BuildCertUtil.cmd

```diff
@@ -37,5 +37,4 @@ echo.
 findstr /ir /c:".*Warning(s)" /c:".*Error(s)" /c:"Time Elapsed.*" "%_buildlog%"
 echo Build Exit Code = %BUILDERRORLEVEL%
 
-endlocal
```

Comment:
It should not. The documentation said it would implicitly call endlocal to each setlocal when a batch file exit.